locales/th/php.json

```diff
@@ -1,40 +1,40 @@
 {
     "accepted": "ข้อมูล :attribute ต้องผ่านการยอมรับก่อน",
-    "accepted_if": "ข้อมูล :attribute ต้องผ่านการยอมรับเมื่อ:otherเท่ากับ:value",
+    "accepted_if": "ข้อมูล :attribute ต้องผ่านการยอมรับ เมื่อ :other เป็น :value",
     "active_url": "ข้อมูล :attribute ต้องเป็น URL เท่านั้น",
     "after": "ข้อมูล :attribute ต้องเป็นวันที่หลังจาก :date",
     "after_or_equal": "ข้อมูล :attribute ต้องเป็นวันที่ตั้งแต่วันที่ :date หรือหลังจากนั้น",
     "alpha": "ข้อมูล :attribute ต้องเป็นตัวอักษรภาษาอังกฤษเท่านั้น",
     "alpha_dash": "ข้อมูล :attribute ต้องเป็นตัวอักษรภาษาอังกฤษ ตัวเลข และ _ เท่านั้น",
     "alpha_num": "ข้อมูล :attribute ต้องเป็นตัวอักษรภาษาอังกฤษ ตัวเลข เท่านั้น",
     "array": "ข้อมูล :attribute ต้องเป็น array เท่านั้น",
-    "ascii": ":Attribute ต้องมีอักขระและสัญลักษณ์ที่เป็นตัวอักษรและตัวเลขคละกันแบบไบต์เดี่ยวเท่านั้น",
+    "ascii": ":attribute ต้องมีอักขระและสัญลักษณ์ที่เป็นตัวอักษรและตัวเลขคละกันแบบไบต์เดี่ยวเท่านั้น",
```

Comment:
Just a notification. [Read up](https://github.com/Laravel-Lang/common/discussions/7) on Laravel's internal mechanics if you haven't encountered them before.

Although, judging by the writing of the characters, in the Thai language letters are not divided into upper and lower case.

Reply:
Thank you. I'm thinking about that as well. But all these strings in English they are `"The :attribute..."` and yes there's no concept of uppercase and lowercase in Thai script.